### DISCUSSION TITLE
Disabled at startup

### DISCUSSION THREAD

**Discussion by Scoubi-7** [blueman-project/blueman#2247] [Q&A]
Hi !

Could you explain me how to disable bluetooth at startup ?
I would like bluetooth to start at startup but be disabled and if I want to activate it I click on the notification icon
I'm under Mint/Cinnamon

**Comment by cschramm** [comment]
blueman disables Bluetooth by soft-blocking it with RFkill and provides a way to re-enable it as long as it detects that state. The RFkill system generally keeps it state throughout reboots.

You probably want to do something like `rfkill block bluetooth` in something like `/etc/rc.local`.

An alternative to RFkill is to unpower the adapters. To avoid powering adapters, disable `AutoEnable` in `/etc/bluetooth/main.conf`.
